rust/rebase/src/witness/dns.rs

```diff
@@ -1,22 +1,22 @@
-use crate::schema::schema_type::{SchemaError, SchemaType};
-use crate::signer::signer::{SignerError, SignerType, DID as SignerDID};
-use crate::witness::{
-    signer_type::SignerTypes,
-    witness::{Generator, Proof, Statement, WitnessError},
+use crate::{
+    schema::schema_type::{SchemaError, SchemaType},
+    signer::signer::{SignerError, SignerType, DID as SignerDID},
+    witness::{
+        signer_type::SignerTypes,
+        witness::{Generator, Proof, Statement, WitnessError},
+    },
 };
 use async_trait::async_trait;
 use chrono::{SecondsFormat, Utc};
+use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 use serde_json::json;
 use ssi::{one_or_many::OneOrMany, vc::Evidence};
 use std::collections::HashMap;
 use url::Url;
 
-// TODO: Move to own dir, maybe w/ schema?
-// TODO: Add Serde
-// TODO: Support the more specific TZProfiles attestation. Requires TZProfiles specific text.
-
-#[derive(Clone, Deserialize, Serialize)]
+#[derive(Clone, Deserialize, JsonSchema, Serialize)]
+#[serde(rename = "claim")]
```

Comment:
Is the name of the type purely for the JSON schema?

Reply:
It came up when trying to validate schemas, all other schema key strings were lower cased.